cibyl/cli/parser.py

```diff
@@ -66,40 +66,61 @@ def __init__(self, ci_args: dict = None, app_args: dict = None):
         self.app_args = app_args
         if not app_args:
             self.app_args = {}
-        self.argument_parser = argparse.ArgumentParser()
+        # parser for model-related arguments
```

Comment:
What's the difference between the model and app parser? (WDYM with Model related arguments vs app?)

Reply:
The model arguments are the ones that are added from the Model API's: --jobs, --builds, --tests, --ip-version, ...

Reply:
Another question: will we update the `parser` section of the docs? https://cibyl.readthedocs.io/en/latest/parser.html

Reply:
Yes, that's a good point, I will update several pages of the documentation (parser, usage, and probably others) after we are done with this change, since the details of the implementation might vary after the review.

Reply:
Excellent, Joan.